test/clearingHouse/ClearingHouse.badDebt.test.ts

```diff
@@ -56,30 +61,44 @@ describe("ClearingHouse badDebt", () => {
         await collateral.mint(bob.address, parseUnits("100", decimals))
         await deposit(bob, vault, 100, collateral)
 
+        // prepare collateral for carol
+        await collateral.mint(carol.address, parseUnits("1000000", decimals))
+        await deposit(carol, vault, 1000000, collateral)
+
         // alice add liquidity
         await addOrder(fixture, alice, "500", "50000", lowerTick, upperTick, false)
+
+        // initiate both the real and mocked timestamps to enable hard-coded funding related numbers
+        // NOTE: Should be the last step in beforeEach
+        await initiateBothTimestamps(clearingHouse)
     })
 
     describe("close/reduce position when bad debt", () => {
-        describe("taker has long position and market price becomes lower than index price", async () => {
+        describe("taker has long position and market price becomes lower than mark price", async () => {
             beforeEach(async () => {
                 // bob long base Token with 8x leverage
                 // bob open notional: -800
-                // bob position size: 7.866
+                // bob position size: 7.866265610482054835
                 await q2bExactInput(fixture, bob, "800", baseToken.address)
-                // market price = index price = 103.222
+
+                // To ignore founding payment
                 await syncIndexToMarketPrice(mockedBaseAggregator, pool)
+                // bob long base token for a 30 mins to manipulate mark price
+                await forwardBothTimestamps(clearingHouse, 1800)
+                // carol short base token that causing bob has bad debt(if he close his position)
+                await b2qExactInput(fixture, carol, "5000", baseToken.address)
 
-                // alice short base token that causing bob has bad debt(if he close his position)
-                await b2qExactInput(fixture, alice, "5000", baseToken.address)
+                // markPrice: 103.199929556499904188
+                // marketPrice: 0.828815379024741938
 
                 // bob's account value is greater than 0 bc it's calculated by index price
-                // bob's account value: 100 + 7.866 * 103.222 - 800 = 111.944
-
-                expect(await clearingHouse.getAccountValue(bob.address)).to.be.eq("111974414000000000000")
+                // bob's account value: 100 + 7.866265610482054835 * 103.199929556499904188 - 800 = 111.7980568745
+                expect(await clearingHouse.getAccountValue(bob.address)).to.be.gt("0")
```

Comment:
If we know the exact number, suggest expecting it instead of using `gt`

Reply:
fixed in 558a8d2db7c402d809df779c2a463cd5def4c129